ymcatwincities/openy
# Ability to set default selection for popup location picker and fix cookie set os-30

Provide a way to stay informed of Y events happening in some area.

## Steps for review

- Install from profile or update existing site
- Visit /schedules page
 In case no cookie "openy_preferred_branch" not set should display popup with 'all' radio checked 
- Login to system 
-  Visit /admin/config/content/location-popup
- Should be able to select default location popup
- Save config
- Visit /schedules page
 In case no cookie "openy_preferred_branch" not set should display popup with saved radio checked
-  Click on "Set Location" button
- cookie "openy_preferred_branch" should be set
- Visit /schedules page
Popup should not be displayed. 

Please review, thanks
